src/components/body/BodyRow.ts

```diff
@@ -38,22 +38,19 @@ import { DataTableBodyCell } from './BodyCell';
       </div>
     </div>
   `,
-  directives: [ DataTableBodyCell ],
-  host: {
-    '[class.active]': 'isSelected'
-  }
+  directives: [ DataTableBodyCell ]
 })
 export class DataTableBodyRow {
 
   @Input() row: any;
 
-  get isSelected() {
-    return this.state.selected &&
-      this.state.selected.indexOf(this.row) > -1;
+  constructor(private state: StateService) {
   }
 
-  constructor(private state: StateService, elm: ElementRef) {
-    elm.nativeElement.classList.add('datatable-body-row');
+  @HostBinding('class.datatable-body-row') isBodyRow: boolean = true;
```

Comment:
Gotcha. I had it like this originally but switched to this given it was faster. Given perf and number of rows its super important to get every inch of perf we can.

I think we should raise this to the ng team for feedback.